samples/bluetooth/direction_finding_connectionless_rx/README.rst

```diff
@@ -0,0 +1,162 @@
+.. _direction_finding_connectionless_rx:
+
+Bluetooth: Direction finding connectionless locator
+###################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52833dk_nrf52833
+
+The sample also requires an antenna matrix when operating in Angle of Arrival mode.
+It can be a Nordic Semiconductor design 12 patch antenna matrix, or any other antenna matrix.
+
+Overview
+********
+
+The direction finding connectionless locator sample application demonstrates Bluetooth LE direction finding reception.
+It uses Constant Tone Extension (CTE), that is received and sampled with periodic advertising PDUs.
```

Comment:
```suggestion
The direction finding connectionless locator sample application uses Constant Tone Extension (CTE), that is received and sampled with periodic advertising PDUs.
```
Moving the first sentence to before the Requirements section.

Reply:
Sentence moved according to suggestion.

Reply:
I see you missed the suggested sentence change, so it looks a bit strange starting a new section with a sentence starting off with "It uses..." :)

Reply:
Ah... I'm sorry. Missing sentence added.